pkg/kubernetes/kube.go

```diff
@@ -953,17 +953,34 @@ func (kc *Client) ValidatePrometheusVolumeClaimTemplatesName(statefulsetName str
 		return errors.Errorf("Prometheus volumeClaimTemplate name changed', got: %v", sfsvolumeClaimTemplatesName)
 	}
 	// Validate Persistent Volume label
-	requiredLabels := []string{"failure-domain.beta.kubernetes.io/zone", "topology.kubernetes.io/zone"}
-	err = kc.validatePVLabels(volumeClaimTemplatesName, requiredLabels)
```

Comment:
Now that we are not using this we need to remove the function because its not used anymore, is it?

Reply:
deleted